statchatui/src/Compare/Compare.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useEffect } from "react";
+import Stats from "../Stats/Stats";
+import "./Compare.css";
+
+const GAME_NAMES = {
+  APEX: "Apex Legends",
+  R6: "Rainbow Six Siege",
+};
+const Compare = ({ signedInUserData, fetchStats }) => {
```

Comment:
There's a lot of similarities between Compare.jsx and Home.jsx!

In software engineering there is a pattern called **composition**. A car isn't built from scratch each time; it's composed of a bunch of common parts (an engine, tires, a steering wheel). 

Ideally, we want to create a few smaller React components and use them as building blocks inside of bigger ones. Here's an example:

```
function PlatformPicker({onChange, value}) {
        return <select
          onChange={onChange}
          name="platform"
          className="input-field"
          value={value}
        >
          <option value="">Select Platform</option>
          <option value="Xbox">Xbox</option>
          <option value="PC">PC</option>
          <option value="PS4">PlayStation</option>
        </select>
}
```

```
// Now you can drop in this code anywhere
<PlatformPicker value="Xbox" onChange={onPlatformChange} />
// ... and do something similar with the other form fields
<GamePicker value="Apex" onChange={onGameChange} />
```